test/metric_learn_test.py

```diff
@@ -861,7 +861,8 @@ def test_deprecation_num_dims_lfda(num_dims):
 
 class TestRCA(MetricTestCase):
   def test_iris(self):
-    rca = RCA_Supervised(n_components=2, num_chunks=30, chunk_size=2)
+    rca = RCA_Supervised(n_components=2, num_chunks=30, chunk_size=2,
```

Comment:
I set the random seed here to pass the test. Note that the random seed was not set before, so I checked that the test passed with a random_state 42 before, it did, and now it passes too